apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabaseBase.cs

```diff
@@ -58,13 +58,10 @@ public abstract class SpannerTestDatabaseBase
     /// </summary>
     public string SpannerDatabase { get; }
 
-    // This is the simplest way of checking whether the environment variable was specified or not.
-    // It's a little ugly, but simpler than the alternatives.
     /// <summary>
-    /// Returns true if the database was created just for this test, or false if the database was an existing one
-    /// specified through an environment variable.
+    /// Returns true if a new database was created just for this test, false otherwise.
     /// </summary>
-    public bool Fresh => SpannerDatabase == GeneratedDatabaseName;
+    public bool Fresh { get; } = false;
```

Comment:
This is a `bool` with a default value of `false`. Do we need to set it explicitly?